diff --git a/httpman.go b/httpman.go
index 6757ab4..1214a74 100644
--- a/httpman.go
+++ b/httpman.go
@@ -15,7 +15,7 @@ import (
 
 // YAMLFILE describes where the HTTP status code YAML file
 // is located. Should be patched per user needs
-const YAMLFILE string = "/etc/httpman_codes.yml"
+const YAMLFILE string = "/your/location/here.yml"
 
 // global map to hold all status code info
 // when YAMLFILE is unmarshaled
